src/skills-builder/skills-builder-steps/select-preferences/JobTitleInstantSearch.jsx

```diff
@@ -41,7 +41,7 @@ const JobTitleInstantSearch = ({ onSelected, ...props }) => {
         )}
       {...props}
     >
-      {hits.map(job => (
+      {!indexUiState.query ? [] : hits.map(job => (
```

Comment:
We now check for empty query here. If the query is empty, we return an empty array